### PR TITLE
Use new patternfly jumplinks.

### PR DESCRIPTION
**Pull request by ssilvert**
## Motivation
ScrollForm should use new patternfly JumpLinks instead of our home-grown logic.

https://www.patternfly.org/v4/components/jump-links

## Brief Description
* Upgraded @patternfly packages.
* Refactored ScrollForm

## Verification Steps
Go to client details.  Links on the right side of the page should work the same as before.

## Checklist:

- [X] Code has been tested locally by PR requester
- [X] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated
- [X] Formatting has been performed via prettier/eslint
- [X] Type checking has been performed via 'yarn check-types'

## Additional Notes
At some point we should also implement the scroll spy capability of JumpLinks.  I tried it out and couldn't get it to work right away with the Grid layout we are currently using in ScrollForm.  The example code for JumpLink uses PageSections so maybe if we converted to that it would work.  But that can be a task for another day.

### CODE REVIEW

File: src/components/scroll-form/ScrollForm.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import React, { Children, useEffect, useState } from "react";
```
Comment (edewit) [inline src/components/scroll-form/ScrollForm.tsx:1]: `useEffect` can be removed

**Review by edewit — APPROVED** [review]
:+1: always good to see so much code removed

Although the new doesn't indicate the active section when scrolling down like the old did

**Comment by ssilvert** [comment]
> 👍 always good to see so much code removed
> 
> Although the new doesn't indicate the active section when scrolling down like the old did

Yeah, I couldn't get the scroll spy feature working but I didn't spend much time on it. Sarah says she thinks she knows how to do it.  It's a beta component after all.  I'm confident we will get it working.

File: src/components/scroll-form/ScrollForm.tsx (hunk 1 above)
Comment (ssilvert) [reply src/components/scroll-form/ScrollForm.tsx:1]: Oops.  Missed this comment.  I'll submit another PR to fix it.

**Pull request merged**